src/pages/about.js

```diff
@@ -21,49 +21,38 @@ class App extends React.Component {
           <p>
             Blacks Who Engineer is a Twitter directory of
             accomplished Black folks in the
-            design industry. It aims to help people find notable and relevant
+            software industry. It aims to help people find notable and relevant
             voices to follow on Twitter by parsing Twitter bios for popular
             keywords.
           </p>
           <p>Here are some things Blacks Who Engineer can help you with:</p>
-          <h2 className={styles.emphasis}>
-            Use{" "}
-            <a href="https://www.proporti.onl/" target="_blank">
-              proporti.onl
-            </a>{" "}
-            to check the ratio of the people you follow on Twitter.
-          </h2>
-          <p>
-            If you're following more men than women, use this project to follow
-            new women and diversify the voices in your feed. Be aware that a
-            feed of white women is not diverse.
-          </p>
           <h2 className={styles.emphasis}>
             If you're a hiring manager, use this project to find candidates.{" "}
           </h2>
           <p>
-            Examine the ratio of senior men to senior women in your
-            organization. Are women of color equally represented? Consider
-            hiring women into promotions above their current role.
+            Examine the diversity of senior engineers and leaders
+            in your organization. Are Black folks, especially
+            Black women, equally represented?  Promote and give
+            growth opportunities to Black engineers.
           </p>
 
           <h2 className={styles.emphasis}>
             If you're organizing a conference, use this project to find
             speakers.
           </h2>
           <p>
-            Ensure that the women's speaking slots are as prominent as the
-            men's. Are women of color equally represented? Consider reaching out
-            to women who have never given a talk before.
+            Ensure that Black folks have prominent speaking slots.
+            Are Black women equally represented? Consider reaching 
+            out to Black folks who have never given a talk before.
```

Comment:
I feel like I might be overusing "Black folks"—I'm _very_ open to suggestions for alternatives to decrease repetitiveness

Reply:
Black folks sounds great to me, doesn't feel like it's being overused!